nss-day-cohort-30/bangazon-api-whimsical-whooping-cranes
# Abbey computer controller branch

# Description
I have created the computer controller with full crud and the testing class, and all tests passed

## Type of change
Please delete options that are not relevant.



# Testing Instructions for Change Made
1. `git fetch --all`
1. `git checkout abbeyComputerControllerBranch`
1. dotnet run
1. verify...


# Checklist:
- [x ] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [ x] I have commented my code, particularly in hard-to-understand areas

LINK FOR TICKET
https://github.com/nss-day-cohort-30/bangazon-api-whimsical-whooping-cranes/issues/9
